charts/pega/KafkaClusterRequirement.md

```diff
@@ -50,4 +50,54 @@ User:\<user-name\> |GROUP  | * | ALL   | ALLOW      | LITERAL
 User:\<user-name\> |CLUSTER  | \<Cluster-Name\> | IDEMPOTENT_WRITE   |ALLOW      | LITERAL
 
 
+### Stream (externalized Kafka service) test example with bitnami kafka
+
+For testing purpose you can install bitnami kafka using below commands 
+
```

Comment:
For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where my-kafka is the namespace of your Kafka service deployment.

Reply:
@taz-pega-work  have done the changes suggested